jax/experimental/jax2tf/jax2tf.py

```diff
@@ -396,6 +396,11 @@ def _unexpected_primitive(p: core.Primitive, *args, **kwargs):
 tf_impl[lax.round_p] = tf.math.round
 tf_impl[lax.nextafter_p] = tf.math.nextafter
 
+def _population_count(x):
+  orig_dtype = x.dtype
+  return tf.bitcast(tf.raw_ops.PopulationCount(x=x), orig_dtype)
```

Comment:
This seems to be the first use of `tf.raw_ops`. Are there any pitfalls to using those ops?

Reply:
There is another use of `tf.raw_ops` in `jax2tf.convert` (however, this is not one of the primitives).
According to [the documentation](https://www.tensorflow.org/api_docs/python/tf/raw_ops) and the associated ["RFC"](https://github.com/tensorflow/community/blob/master/rfcs/20181225-tf-raw-ops.md), `tf.raw_ops` simply exposes all the non-deprecated TF ops for library writers in a backward-compatible way. I thus don't expect this to have any more pitfalls than other elements of the TF API we use.